src/utils/series.ts

```diff
@@ -56,30 +56,20 @@ export const getNextItem = (item: PlaylistItem | undefined, series: Series | und
 /** We need to add episodeNumber and seasonNumber to each media item we got
  *  That will help with further data retrieval
  */
-export const enrichMediaItems = (series: Series | undefined, mediaItems: PlaylistItem[] | undefined): PlaylistItem[] => {
-  let episodes: EpisodeWithSeason[] = [];
-
+export const enrichMediaItems = (series: Series | undefined, mediaItems: { [key: string]: PlaylistItem }): PlaylistItem[] => {
   if (series) {
-    episodes =
-      'seasons' in series
-        ? series.seasons.flatMap((season: Season) =>
-            season.episodes.map((episode) => ({
-              ...episode,
-              season_number: season.season_number,
-            })),
-          )
-        : series.episodes.map((episode) => ({ ...episode, season_number: 0 }));
+    if ('seasons' in series) {
+      return series.seasons.flatMap((season: Season) =>
+        season.episodes.map((episode) => ({
+          ...mediaItems[episode.media_id],
+          seasonNumber: String(season.season_number),
+          episodeNumber: String(episode.episode_number),
+        })),
+      );
+    }
+
+    return series.episodes.map((episode) => ({ ...mediaItems[episode.media_id], seasonNumber: '1', episodeNumber: String(episode.episode_number) }));
```

Comment:
Why did you switch the fallback season / episode number from 0 to 1? I think 0 was used intentionally to identify bad data. Maybe we should use empty string in that case though?

Reply:
I will have a look, i think `0` should work.